conda/recipes/pynvjitlink/meta.yaml

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2018-2024, NVIDIA CORPORATION.
+
+{% set data = load_file_data("pyproject.toml") %}
+
+package:
+  name: pynvjitlink
+  version: {{ data.get("project", {}).get("version", "") }}
+
+source:
+  path: ../../..
+
+build:
+  script:
+    - {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - cuda-version {{ cuda_version }}
+    - cmake >=3.26.4
+    - ninja
+    - sysroot_{{ target_platform }} 2.17
+  host:
+    - libnvjitlink-dev
+    - libnvjitlink-static
+    - cuda-version {{ cuda_version }}
+    - python
+    - pip
+    - scikit-build-core
+  run:
+    - python
+    - numba >=0.57
+    - {{ pin_compatible('cuda-version', max_pin='x', min_pin='x') }}
+
+#test:
```

Comment:
I struggled with getting conda-build to run tests properly. Not even the import worked. However, I was successful in importing the "broken" package manually and running tests locally in `test_conda.sh` so I'm inclined to remove these recipe tests.